tasks/main.yml

```
@@ -2,6 +2,27 @@
 - name: Set version specific variables
   include_tasks: set_vars.yml
 
+- name: Scan sudoers
+  scan_sudoers:
+    output_parsed_configs: True
+    output_raw_configs: False
+  when: sudo_check_if_configured | bool
+
+- name: Compare
+  ansible.utils.fact_diff:
+    before: "{{ ansible_facts['sudoers'].sudoers_files |
+              ansible.utils.remove_keys(target=['include_files']) | to_nice_yaml }}"
+    after: "{{ sudo_sudoers_files | to_nice_yaml }}"
+
+  register: failed_compare
+  when: sudo_check_if_configured | bool
+
+- name: Sudo is already configured
+  meta: end_play
```

```suggestion
  meta: end_host
```
end_play will end the run on all hosts, not just this one - https://docs.ansible.com/ansible/latest/collections/ansible/builtin/meta_module.html "end_play (added in Ansible 2.2) causes the play to end without failing the host(s). Note that this affects all hosts."

---

Done.